client/my-sites/site-settings/settings-discussion/main.jsx

```diff
@@ -27,6 +28,7 @@ const SiteSettingsDiscussion = ( {
 	return (
 		<Main className="settings-discussion site-settings">
 			<DocumentHead title={ translate( 'Site Settings' ) } />
+			<JetpackDevModeNotice />
```

Comment:
The settings `Main` with head, notice, and nav is a good candidate to be extracted into a component.